ddtrace/contrib/httplib/__init__.py

```diff
@@ -9,23 +9,24 @@
     patch(httplib=True)
 
     # Python 2
-    from ddtrace import Pin
     import httplib
     import urllib
 
-    # Use a Pin to specify metadata for all http requests
-    Pin.override(httplib, service='httplib')
     resp = urllib.urlopen('http://www.datadog.com/')
 
     # Python 3
-    from ddtrace import Pin
     import http.client
     import urllib.request
 
-    # Use a Pin to specify metadata for all http requests
-    Pin.override(http.client, service='httplib')
     resp = urllib.request.urlopen('http://www.datadog.com/')
 
+``httplib`` spans do not include a default service name. Before HTTP calls are
+made, ensure a parent span has been started with the service name to be used for
```

Comment:
```suggestion
made, ensure a parent span has been started with a service name to be used for
```

?

(not a strong opinion, just how I read it in my head)